### ISSUE TITLE
Insufficient colour contrast

### ISSUE THREAD

**Issue by ThomasRettig** [kaushikgopal/henry-hugo#1]
Some HTML elements do not have sufficient colour contrast, which prevents accessibility. 

<img width="503" alt="image" src="https://user-images.githubusercontent.com/68767503/142756304-26f9ee48-cd49-40f6-9f7e-8019e3eb8a40.png">

**Comment by kaushikgopal** [comment]
default theme colors have changed curious ff you feel the same about the new colors.

**Comment by ThomasRettig** [comment]
Ahh, it looks much better now! Fantastic! :)
